pkgs/development/compilers/yosys/default.nix

```diff
@@ -18,6 +19,7 @@
 , yosys-bluespec
 , yosys-ghdl
 , yosys-symbiflow
+, enablePython ? false # enable python binding
```

Comment:
Users of tools like `sby` will already need Python anyway; I think it's also enabled in the `nextpnr` build. We can probably go ahead and enable this safely, I think?

Reply:
I don’t think SBY (or any YosysHQ project) relies on the Python binding; it is written in Python but connects to Yosys in other ways. In any case I am not opposed to enabling the binding by default, the heavy boost dependency is the only downside I think.

Reply:
I know it doesn't rely on the python binding, I just meant to say "most users will practically have python in their closure already" for reasons like that, which means this is a very small thing to add, relatively.

Reply:
Ah, yeah, didn't occur to me to think of users for who Python isn't pulled in one way or other already